README.md

```diff
@@ -0,0 +1,150 @@
+# vllm-orchestrator-gateway
+
+This gateway service enables controlled interaction with the FMS Guardrails Orchestrator by enforcing stricter access to its exposed endpoints. It provides fine-grained control over the routes, applying detectors to ensure secure and regulated interactions. Currently, it supports the chat completions endpoint.
```

Comment:
Maybe:

This gateway service enables controlled interaction with the FMS Guardrails Orchestrator by enforcing stricter access to its exposed endpoints. It provides a mechanism of configuring fixed detector pipelines, and then provides a unique `/v1/chat/completions` endpoint per configured detector pipeline. This allows for drop-in replacement of an unguardrailed chat completions model with a guardrailed one.

Reply:
sounds good to me, updating now!